vtm/src/org/oscim/layers/tile/buildings/BuildingLayer.java

```diff
@@ -248,6 +252,15 @@ public void complete(MapTile tile, boolean success) {
             get(tile).resetBuckets(null);
     }
 
+    protected String getValue(MapElement element, String key) {
+        return element.tags.getValue(getKeyOrDefault(key));
+    }
```

Comment:
This method seems superfluous, can stay with current workflow adding the `getKeyOrDefault` for the keys.

Reply:
On second thought can use it, as seems more clear and can change the result in one place.